Tests/StarknetTests/Accounts/AccountTest.swift

```diff
@@ -2,6 +2,8 @@ import XCTest
 
 @testable import Starknet
 
+@available(macOS 13.0.0, *)
+@available(iOS 16.0.0, *)
```

Comment:
As mentioned above, not needed if you bump the requirements in Package.swift. Same applies for everywhere else you've added `@available`.